plugins/survey-resources/src/plugin.ts

```diff
@@ -13,7 +13,20 @@
 // limitations under the License.
 //
 
-import survey, { surveyId } from '@hcengineering/survey'
-import { mergeIds } from '@hcengineering/platform'
+import survey, {
+  type Checkboxes,
+  type QuestionDataEditorComponentTypeRef,
+  type RadioButtons,
+  surveyId
+} from '@hcengineering/survey'
+import { mergeIds, type Resource } from '@hcengineering/platform'
+import { type ComponentType } from 'svelte'
 
-export default mergeIds(surveyId, survey, {})
+export default mergeIds(surveyId, survey, {
+  component: {
+    OptionsQuestionDataEditor: '' as QuestionDataEditorComponentTypeRef<Checkboxes | RadioButtons>,
+    SurveyCreator: '' as Resource<ComponentType>,
+    SurveyNamePresenter: '' as Resource<ComponentType>,
+    QuestionCollectionEditor: '' as Resource<ComponentType>
```

Comment:
Technically, `Resource<ComponentType> === AnyComponent`, but `AnyComponent` is declared in `@hcengineering/ui`, which it's not publishable (and `Resource` and `ComponentType` are declared in `svelte`), which causes transpilation errors saying something like `Cannot name an inferred type of exported default without referencing @hcengineering/ui/src/...`